README.md

```diff
@@ -49,79 +50,123 @@ for _ in range(1000):
     observations, _, _, _ = env.step(agent_actions)
 ```
 
-## Setup
+# Contents
+1. [Documentation](#Documentation)
+1. [Setup](#Setup)
+    + [Installation](#Installation)
+    + [Running](#Running)
+1. [Examples](#Examples)
+    + [Usage](#Usage)
+    + [Reinforcement Learning](#Reinforcement-Learning)
+1. [Task Environments](#Task-Environments)
+1. [CLI Tool](#CLI-Tool)  
+    + [CLI Usage](#CLI-Usage)
+    + [CLI Examples](#CLI-Examples)
+1. [Visualizing Observations](#Visualizing-Observations)
+1. [PyMARL and MALib](#PyMARL-and-MALib)
+1. [Containers](#Containers)
+    + [Docker](#Docker)
+    + [Singularity](#Singularity)
+1. [Troubleshooting](#Troubleshooting)
+    + [General](#General)
+    + [SUMO](#SUMO)
+1. [Bug Reports](#Bug-Reports)
+1. [Contributing](#Contributing)
+1. [Citing](#Citing)
+
+# Documentation
+Documentation is available at [smarts.readthedocs.io](https://smarts.readthedocs.io/en/latest).
 
+# Setup
+### Installation
 ```bash
-# For Mac OS X users, make sure XQuartz is pre-installed as SUMO's dependency
-
-# git clone ...
+git clone https://github.com/huawei-noah/SMARTS.git
 cd <path/to/SMARTS>
 
-# Install the system requirements, then follow the instructions for setting up the SUMO_HOME environment variable. You may use the `-y` option to enable automatic assumption of "yes" to all prompts to avoid timeout from waiting for user input. 
+# For Mac OS X users, ensure XQuartz is pre-installed.
+# Install the system requirements, then follow the instructions for setting up the `SUMO_HOME` environment variable. You may use the `-y` option to enable automatic assumption of "yes" to all prompts to avoid timeout from waiting for user input. 
 bash utils/setup/install_deps.sh
 
-# verify sumo is >= 1.5.0
-# if you have issues see ./doc/SUMO_TROUBLESHOOTING.md
+# Verify sumo is >= 1.5.0.
```

Comment:
I think we have been using 1.10.0 recently.

Reply:
This step is now removed with installation of sumo via `pip install eclipse-sumo==1.10.0` .